### PR TITLE
chore: remove master references in eigen

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **chore**

This PR is a follow-up from https://github.com/artsy/README/issues/427 and https://github.com/artsy/README/issues/386

### Description

- Remove master mentions in the project where it's possible
- Some archived/deprecated repos that we don't own still use master as default branch, we decided to leave those until we agree on best practice

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Remove artsy internal master branch references - mounir

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (Remove artsy internal master branch references - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e534ec04f1fb764ac61a252b01fb1efbc933fd6e
</p>

File: src/lib/Scenes/Artwork/Components/ImageCarousel/FullScreen/ImageCarouselFullScreen.tsx [hunk 1]
```diff
@@ -35,13 +35,13 @@ export const ImageCarouselFullScreen = () => {
 
   const zoomViewRefs: ImageZoomView[] = useMemo(() => [], [])
 
-  const masterOpacity = useAnimatedValue(1)
+  const mainOpacity = useAnimatedValue(1)
```
Comment (dimatretyak) [inline src/lib/Scenes/Artwork/Components/ImageCarousel/FullScreen/ImageCarouselFullScreen.tsx:38]: I think there was no need to rename here

**Review by dimatretyak — APPROVED** [review]
Good job. Thanks for the quick changes

File: src/lib/relay/middlewares/metaphysicsMiddleware.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export function metaphysicsExtensionsLoggerMiddleware() {
       const requests = res.json.extensions?.requests
 
       if (requests && console.groupCollapsed) {
-        // See: https://github.com/artsy/metaphysics/blob/master/src/lib/loaders/api/extensionsLogger.ts
+        // See: https://github.com/artsy/metaphysics/blob/main/src/lib/loaders/api/extensionsLogger.ts
```
Comment (pvinis) [inline src/lib/relay/middlewares/metaphysicsMiddleware.ts:20]: I wonder if MP is using main already?

File: src/lib/Scenes/Artwork/Components/ImageCarousel/FullScreen/ImageCarouselFullScreen.tsx (hunk 1 above)
Comment (pvinis) [reply src/lib/Scenes/Artwork/Components/ImageCarousel/FullScreen/ImageCarouselFullScreen.tsx:38]: true but it doesn't hurt to get rid of `master` everywhere 👍

**Review by pvinis — APPROVED** [review]
nice

File: src/lib/Scenes/Artwork/Components/ImageCarousel/FullScreen/ImageCarouselFullScreen.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/lib/Scenes/Artwork/Components/ImageCarousel/FullScreen/ImageCarouselFullScreen.tsx:38]: I actually thought about this change for a while and spoke about it and was asked to change it

File: src/lib/relay/middlewares/metaphysicsMiddleware.ts (hunk 2 above)
Comment (MounirDhahri) [reply src/lib/relay/middlewares/metaphysicsMiddleware.ts:20]: yes this is a nice comment. I asked devon to do it once he deploys his changes here https://github.com/artsy/metaphysics/pull/3595

File: src/lib/Scenes/Artwork/Components/ImageCarousel/FullScreen/ImageCarouselFullScreen.tsx (hunk 1 above)
Comment (dimatretyak) [reply src/lib/Scenes/Artwork/Components/ImageCarousel/FullScreen/ImageCarouselFullScreen.tsx:38]: Got it. Thanks for the clarification

File: src/lib/relay/middlewares/metaphysicsMiddleware.ts (hunk 2 above)
Comment (MounirDhahri) [reply src/lib/relay/middlewares/metaphysicsMiddleware.ts:20]: he already did it, and he will be changing the update-eigen-query as well script

**Comment by MounirDhahri** [comment]
#squashongreen

**Pull request merged**